pixart.js

```diff
@@ -1 +1,56 @@
+$("document").ready(function(){
+  var brushInput;
+//add event listener to button
+  $("#set-color").on("click", function(evt){
```

Comment:
if listen for `on("submit", function)` will actually submit both click and enter events.
[submit](https://api.jquery.com/submit/)